vignettes/intro_htmlwidgets.Rmd

```diff
@@ -120,6 +123,8 @@ data = sampleData,
 sparklinesLine(color = "#56b45d"),
 sparklinesSpots(style = list(fill = "#56b45d"))
 )
+=======
+reactWidget('reactSparklines', 'output', SparklinesComponents, {});
 ```
```

Comment:
@alandipert I don't think your addition here was intentional?

Reply:
It wasn't, but fixed on `enhancements` 👍